wiki/People/Beatmap_Nominators/General_Information/en.md

```diff
@@ -1,57 +1,46 @@
+- [Report a Qualified beatmap here!](https://osu.ppy.sh/community/forums/topics/447428): If you see a Qualified map breaking one of the rules, or feel that it could still use substantial improvement for whatever reason, feel free to report it here by linking the map and giving a brief outline of the problems you see.
+- For beatmaps with subjective quality concerns that don't necessarily break the rules and guidelines of the Ranking Criteria, but which you feel which make the beatmap in question unfit for the Ranked section, the [beatmap veto](/wiki/People/Beatmap_Nominators/Beatmap_Veto) allows you to withhold a beatmap from Qualification.
+- [Mapset Verifier](https://osu.ppy.sh/community/forums/topics/943895): A tool built by NAT member [Naxess](https://osu.ppy.sh/users/8129817). This tool checks for a wide range of unrankable issues, as well as features a snapshot feature to easily compare versions of maps which is beneficial when rechecking. **This should not replace checking a beatmap yourself**, but instead be used as a tool to help double-check nothing slipped by.
 
-First off, take note of the [Beatmap Nominator Rules](/wiki/Beatmap_Nominator_Rules), which outlines both how to undertake various actions such as iconing maps, as well as how not to do things - you would want to avoid breaking these in any situation, so be sure to read them carefully! You can always ask a member of the [Quality Assurance Team](/wiki/People/Quality_Assurance_Team) if you are unsure about something, and we will be happy to help.
+## Beatmap Nominator abilities
 
-Next, there's the [Code of Conduct for Mapping and Modding](/wiki/CoC). As a Beatmap Nominator, you are expected to be an exemplary figure of the community, someone others look up to and respect, so misconduct may get you reprimanded or even expelled. This also includes parts of the ranking process as well - do not go around nominating maps in which the mapper has ignored mods, for example.
+Nominations and resetting nominations are the evidence of a Beatmap Nominator's efforts when checking maps, indicating a map's progress through the ranking proccess. As a Beatmap Nominator you have access to these tools and must use them wisely.
 
-Moreover, as a Beatmap Nominator, you have the opportunity to join a Nominator Subdivision. These provide an opportunity to collaborate in small groups, discuss potential problems with your peers, or gain a fresh perspective. Joining a Subdivision is highly recommended for new Beatmap Nominators to ensure you can always ask for help and guidance when necessary. You can find more information on how these work in this document.
+### Nominate ![nominate button](/wiki/shared/icon/nominate.png)
```

Comment:
dont inline these images, they can go below header (the only images we ever inline to titles are small things like mode icons). also move to this-folder/img/nominate.png and same wtih the others
```suggestion
### Nominate

![nominate button](img/nominate.png)
```